CMakeLists.txt

```diff
@@ -19,12 +19,8 @@ endif()
 
 add_subdirectory(witmotion-uart-qt)
 
-#set(CMAKE_AUTOMOC ON)
-#qt5_wrap_cpp(MOC_SOURCES include/witmotion_ros.h)
 include_directories(include)
-include_directories(/opt/ros/humble/include/tf2_geometry_msgs)
```

Comment:
Removed any hardcoded path to ensure it will compile on all ROS2 deployments like robostack, source builds, etc.